data/json/monsters/zed-animal.json

```
@@ -470,7 +470,7 @@
     "type": "MONSTER",
     "name": { "str": "woodland wight" },
     "looks_like": "mon_deer",
-    "description": "Pale, stumbling white-tailed deer with blackened eyes and drooling slime down its neck.  Its legs twist and bend in strange, haphazard directions, but with unnatural strength and speed.  Preys on coyotes, wolves, and giant spider mutants.",
+    "description": "A pale, stumbling, white-tailed deer, rivulets of slime running down its neck. Its limbs twist and bend in unnatural, haphazard directions, and yet it moves with uncanny strength and speed.  Thanks to the Cataclysm, it has now turned the tables, and what was once prey, now itself preys on coyotes, wolves, and giant mutated spiders.",
```

```suggestion
    "description": "A pale, stumbling, white-tailed deer, rivulets of slime running down its neck.  Its limbs twist and bend in unnatural, haphazard directions, and yet it moves with uncanny strength and speed.  Thanks to the Cataclysm, it has now turned the tables; what was once prey, now itself preys on coyotes, wolves, and giant mutated spiders.",
```
Space is definitely needed. Latter part is stylistic.

---

Oh I thought this should have been caught by JSON validation.